docs/source/cookbook/ccdc.rst

```diff
@@ -1,10 +1,475 @@
+
+**Chi Square Probability**
+
+The Chi-Square test will test if an observation is part of the general statistical distribution of the time-series. A low value of Chi-Square probability will favor the rejection of the null-hypothesis (i.e. being part of the statistical distribution), therefore flagging it as possible change. Ultimately, a lower value leads to more breaks detected, which favors omission over commission error.
+A high value allows for more noise in the time-series, and less changes will be detected, therefore lowering the commission error rate.
+
+**Min Number of Years Scaler**
+
+This parameter determines the minimum length of any inner temporal segment.
+
+**LAMBDA**
+
+The lambda parameter is part of the LASSO regression used for the modelling of the time-series. It is used to generalize the model and thereby improving its predictive power. More specifically, it is controlling the weight of each of the parameters, and might result even in the annulation of some of the parameters. In practical terms, an initially 3rd order harmonic model, might shrink to a 1st order harmonic, if this provides the best generalized fit. Setting lambda to 0 will lead to a regular Ordinary-Least-Square regression, not providing any generalization. Instead, a higher value will provide a more generalised model. If lambda is set too high, the model will underfit, which also not wanted. Since a value of 20 has been found to provide a generally good performance, the sweet spot of neither over- nor underfitting will be around this number.
+
+**Max iterations**
+
+Those are the iterations for the maximum number of runs for LASSO regression convergence. If set to 0, regular OLS is used instead of LASSO.
+
+On-the-fly Pixel analysis
+-------------------------
+
+Click on the :icon:`fa fa-chart-area` button to start the plotting tool (1). Move the pointer to the main map, the pointer will be transformed into a :icon:`fa fa-plus` (2). Click anywhere in the AOI to plot data for this specific location in the following popup window.
+
+The plotting area (3) is dynamic and can be customized by the user.
+
+You can select the observation feature by selecting one of the available measures in the dropdown selector in the top left corner (4). The available bands are the same as the described previously.
+
+Using the slider (5), the temporal width displayed can be changed. It cannot exceed the start and/or end date of the time series.
+
+On the main graph, the orange lines shows the CCDC modelled time-series. Each of the blue points represents an actual observation. You can both hover over the point or the line to let the tooltip describe the value and date of the observation, as well as the model values and the temporal extent of the specific segment.
```

Comment:
on the example image the orange area around the line is not always the same width, is it the error ?

Reply:
for the segments it is, it is the rmse